show/interfaces/__init__.py

```diff
@@ -538,7 +537,7 @@ def counters(ctx, verbose, period, interface, printall, namespace, display):
             cmd += " -s {}".format(display)
         if namespace is not None:
             cmd += " -n {}".format(namespace)
-
+        click.echo("\n\nReminder: Please check internal links for any CRC error with '-d all' option\n\n")
```

Comment:
Can we print this only for multi-asic platform? Also, it may be better to display this at the end of the command output.

Might be better to show the entire command in the message - something like "Please execute "show interface counters -d all" to include internal links". 

Also, since this will help identify CRC errors as well as ASIC drops, better not to mention "CRC error"